### PR TITLE
Added info about v21 to the README

### PR DESCRIPTION
**Pull request by fballiano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
 
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 is now the minimum version
```
Comment (ADDISON74) [inline README.md:271]: PHP 8.1 as minimum required version

File: README.md [hunk 2]
```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
...
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730), if you have old modules that still use those classes, please use the 'shell/rename-mysql4-class-to-resource.php` to convert them
```
Comment (ADDISON74) [inline README.md:276]: Removed all deprecated Mysql4_ classes (#2730)**. If you have installed old modules/extensions that use such classes then you must run in CLI /{OpenMage_ROOT}/shell/rename-mysql4-class-to-resource.php to convert them. Please backup all files before running the script**

**Review by ADDISON74 — CHANGES_REQUESTED** [review]

**Comment by ADDISON74** [comment]
It was absolutely necessary to keep a list of the differences between the main and next branches. You have certainly kept records and you can check if there are others. Those who use OM must be prepared for changes in the future, I think that very few use the next branch in testing. This list will help them.

**Comment by fballiano** [comment]
github is doing it already https://github.com/OpenMage/magento-lts/pulls?q=is%3Apr+is%3Amerged+base%3Anext

File: README.md [hunk 3]
```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
...
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 as minimum required version
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730), if you have old modules that still use those classes, please use the 'shell/rename-mysql4-class-to-resource.php` to convert them
```
Comment (fballiano) [inline README.md:276]:
```suggestion
- Removed all deprecated Mysql4_ classes (#2730), if you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. backup all files before running the script.
```

File: README.md [hunk 4]
```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
...
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 as minimum required version
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730), if you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. backup all files before running the script.
```
Comment (ADDISON74) [inline README.md:276]: "backup" with capital B :)

**Review by ADDISON74 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (ADDISON74) [reply README.md:276]: I would split the paragraph in two parts 

Removed all deprecated Mysql4_ classes (#2730)**. I**f you ...

File: README.md (hunk 3 above)
Comment (pquerner) [reply README.md]: That script only works on systems with "gnu grep" installed in $PATH, should probably be noted. Although.. developers probably know..

File: README.md (hunk 4 above)
Comment (fballiano) [inline README.md:276]:
```suggestion
- Removed all deprecated Mysql4_ classes (#2730).
If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.
```

**Review by ADDISON74 — DISMISSED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Comment by fballiano** [comment]
that point is now the only one in the entire readme that has a ".", it doesn't look good to me, that's why I preferred one sentence. I don't think we've to say to backup, still.

**Comment by ADDISON74** [comment]
Often when things don't go well people tend to blame others. By mentioning to backup the files is the way to avoid further discussions.

**Comment by fballiano** [comment]
it's written everywhere to backup when updating. anyway. I still think it looks off to have only one dot in the lists, but ok

**Comment by ADDISON74** [comment]
You can remove the last dot (after the "script" word) to keep the same format. All the others are fine.

File: README.md [hunk 5]
```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
...
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 as minimum required version
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730). If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.
```
Comment (ADDISON74) [inline README.md:276]: "If there are any old modules/extensions in your installation that use such classes, you must run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script"

The wording sounds better. I removed the dot at the end of the bullet in order to keep the same format used in the readme file.

OFFTOPIC: the script name it is too long compared to the other existing ones. rename-mysql4 or mysql4 were more than enough.

**Review by ADDISON74 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (fballiano) [inline README.md:276]:
```suggestion
- Removed all deprecated Mysql4_ classes (#2730). If there are any old modules/extensions in your installation that use such classes, you must run `shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script
```

**Review by ADDISON74 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (ADDISON74) [reply README.md]: The use of the word "rename" in the name of the file is not suitable for the action that the script does. In addition, the name must be as short as possible. If we look in the /shell directory we see how simple the file names are (indexer.php, log.php). I think the most suitable would be mysql4.php.

Comment (fballiano) [reply README.md]: mmm no i don't agree, mysql4.php really would be impossible to understand, also, this is a one shot usage file and it's pontentially dangerous, the filename should be at least understandable

**Comment by ADDISON74** [comment]
**Offtopic:** #3146 - This merged PR should be in the main branch. It is a very useful feature. Moving a website to the next branch will involve effort. Jquery is 3.7 instead of 1.12, used in most existing extensions. this in addition to bringing extensions to PHP 8.2. Therefore, until the transition is made, certain PRs should be merged in the main branch, as is the one mentioned by me.

**Pull request merged**